src/libraries/System.Net.Security/src/System/Net/Security/TlsCipherSuiteData.cs

```diff
@@ -24,8 +24,9 @@ static TlsCipherSuiteData()
                 s_tlsLookup.Count == LookupCount,
                 $"Lookup dictionary was of size {s_tlsLookup.Count} instead of {LookupCount}");
 
-            foreach (TlsCipherSuite val in Enum.GetValues(typeof(TlsCipherSuite)))
+            foreach (object? value in Enum.GetValues(typeof(TlsCipherSuite)))
```

Comment:
This doesn't seem like a nice pattern. Is this really how you have to enumerate enum values now?

Reply:
https://github.com/dotnet/csharplang/issues/3214

Reply:
I almost want to suggest to shutting off `#nullable` around this loop, that's how disappointed I am with this result.

Reply:
That's what dotnet/csharplang#3214 is about fixing.

Reply:
I meant in the meantime, until it is fixed in the compiler.

Reply:
We should annotate things the way everyone else will need to.  I don't think we'll advocate wrapping every foreach in a #nullable disable.  Either this gets fixed in the compiler, or we may need to make the `IEnumerator` interface itself oblivious.